code/modules/clothing/suits/armor.dm

```diff
@@ -585,7 +577,7 @@
 		melee = 50,
 		bullet = 50,
 		energy = 50,
-		bomb = 25,
+		bomb = 20,
```

Comment:
I don't see why this nerf is necessary, personally. The crusader armor is already on the lower end due to its raised stiffness.

Reply:
I hate 5%. That is all.

Reply:
I'll put it back.

Reply:
You could raise it to 30 too if you want